### ISSUE TITLE
Document boilerplate actions needed for game

### ISSUE THREAD

**Issue by drewish** [MikeTaylor/scottkit#6]
As someone who's ignorant about the conventions of these kind of games (I played a couple back the mid-80s and found the guessing at what words you needed to type rather frustrating). It seems like the engine supports quite a few actions but you have to explicitly add support? 

One example is what how do you exit? I've just been hitting ctrl+c but seems like you'd want to be able to give them an elegant way to quit.

**Comment by drewish** [comment]
Oh looking back I noticed https://github.com/MikeTaylor/scottkit/blob/master/data/tutorial/tutorial.md#stage-3-source exposes a few of these action explicitly:
```
action score: score
action inventory: inventory
action look: look
```
I suppose that's kind of what I'm looking for but with a bit more of an explanation of what it's doing there.

**Comment by MikeTaylor** [comment]
Yep, good call. What you need in this case is:
```
action quit:
  print "Goodbye"
  game_over
```
But it would make sense to document this stuff somewhere. (And I admit that myself I just use CTRL-D or CTRL-C.(

Or maybe make a library of standard actions that you can include. Although before I do that I'd need to make a file-inclusion facility (which is on to TODO list anyway). I wish this rather primitive bug-tracker had a way to set up dependencies of one issue on another.

**Comment by drewish** [comment]
(What I tend to do with dependent issues is just add a "Blocked" label and then edit the description to link to the upstream issue.)

I think just giving a file of examples would be great. People could pick an choose the bits that are relevant.

**Comment by MikeTaylor** [comment]
I think this need is met by today's blog-post, [Boilerplate for ScottKit games: standard actions](https://reprog.wordpress.com/2017/10/25/boilerplate-for-scottkit-games-standard-actions/). I you agree, please close this issue.

(You probably already noticed, but the present series of blog-posts is linked from the top-level `README.md`. Once the set is complete, I _might_ translate into Markdown and keep them all inside this git repo.)

**Comment by MikeTaylor** [comment]
Thanks!

We're getting near the end of the sequence of tutorial posts now. I envisage three more:
* counters
* saved rooms
* (non-technical) some aspects of game design

**Comment by drewish** [comment]
Yeah sounds good. I’m really curious about counters as a way to have some kind of conversation trees. Now that I got the slack part working I need to spend some time on my game again ;)

**Comment by MikeTaylor** [comment]
You have it working in Slack? Where? Can I see it?

(I don't see the code in https://github.com/drewish/scottbot yet.)

**Comment by drewish** [comment]
The code is up there. It's actually very little code at this point: https://github.com/drewish/scottbot/blob/master/scottbot.rb

I don't have it running on a server (just been running it from my laptop) at this point so sort of hard to show off.

**Comment by MikeTaylor** [comment]
Sweet! And it's working?

I'd love to see this, so if putting it on a server is easy, do let me know when it's up.
